clang/include/clang/Sema/Sema.h

```diff
@@ -5153,6 +5153,12 @@ class Sema final : public SemaBase {
     return ExprEvalContexts.back();
   };
 
+  ExpressionEvaluationContextRecord &parentEvaluationContext() {
+    assert(ExprEvalContexts.size() >= 2 &&
+           "Must be in an expression evaluation context");
+    return ExprEvalContexts[ExprEvalContexts.size() - 2];
+  };
+
   const ExpressionEvaluationContextRecord &parentEvaluationContext() const {
```

Comment:
You should implement const overload in terms of non-const overload you just added via `const_cast` on `this` to avoid duplication.